src/main/java/com/github/fairit/destinypro/dto/playerapi/Response.java

```diff
@@ -1,4 +1,4 @@
-package com.github.fairit.destinypro.dto.player.api;
+package com.github.fairit.destinypro.dto.playerapi;
```

Comment:
inconsistent with other changes - I'd change this back to `player.api`

Reply:
done